frontend/app/templates/components/zoom_slider.html

```diff
@@ -0,0 +1,5 @@
+<label for="{{ ::sliderId }}" class="hidden-for-sighted"
+  ng-bind="::labelText"></label>
+<input type="range" id="{{ ::sliderId }}"
+  min="{{ ::minValue }}" max="{{ ::maxValue }}"
+  value="{{ selectedScaleIndex + 1 }}">
```

Comment:
N.B. we don't use `ng-model` as it doesn't play well with `input[type="range"]`. See https://github.com/angular/angular.js/issues/6726